Package.swift

```diff
@@ -0,0 +1,51 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let package = Package(
```

Comment:
Is it really necessary to put this file into repository root?

Reply:
> Packages are Git repositories, tagged with semantic versions, containing a Package.swift file at their root.

From here: https://github.com/apple/swift-package-manager/blob/main/Documentation/Usage.md

@alalek Unfortunately it's not possible to put the `Package.swift` file elsewhere. Alternatively, we could create another repository like `opencv/opencv-swift` or `opencv/opencv-swift-package` for the Swift Package that is periodically kept up to date with the parent repo. That might also solve the "chicken and the egg" xcframework checksum problem as well.

Reply:
We usually do not store package-specific information in the root of this repository.

> we could create another repository like opencv/opencv-swift or opencv/opencv-swift-package for the Swift Package

I believe this should work.

There is an example for "opencv-python" PyPi packages: https://github.com/skvark/opencv-python
It is not the same (it is not the package itself), just contains build scripts / dependencies settings / other package manager requirements.

---

BTW, Do you have a simple demo example "project" which uses the package? (besides of builtin tests). It is possible to put it into `samples/swift`?